battlebots/database/access.py

```diff
@@ -37,7 +37,8 @@ def remove_bot(user, botname):
                         % (user.nickname, botname, code_dir))
         pass
 
-    bot = session.query(Bot).filter_by(user=user, name=botname).one_or_none()
```

Comment:
(2) was the option I was aiming for in this PR, but this seems to have some problems with code used by both the Flask app and the ranker app (see https://github.com/ZeusWPI/aichallenge/pull/85#discussion-diff-58952867)